### ISSUE TITLE
"Error Occurs"

### ISSUE THREAD

**Issue by stealthrabbi** [vra/bing_brush#2]
I am currently getting this error when trying to use Bing Brush. I have Debug logging on but not many details.

```
2024-01-26 18:12:30,252 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): www.bing.com:443
2024-01-26 18:12:30,538 - urllib3.connectionpool - DEBUG - https://www.bing.com:443 "POST /images/create?q=realistic%20Barney%20the%20Dinosaur%20Scuba%20Diving%20in%20a%20barrier%20reef%0A&rt=4&FORM=GENCRE HTTP/1.1" 200 129189
==> Generating...
==> Your boosts have run out, using the slow generating pipeline, please wait...
2024-01-26 18:12:30,725 - urllib3.connectionpool - DEBUG - https://www.bing.com:443 "POST /images/create?q=realistic%20Barney%20the%20Dinosaur%20Scuba%20Diving%20in%20a%20barrier%20reef%0A&rt=3&FORM=GENCRE HTTP/1.1" 200 129145
==> Error occurs, please submit an issue at https://github.com/vra/bing_brush, I will fix it as soon as possible.

```

**Comment by stealthrabbi** [comment]
does the cookie need to be regenerated?

**Comment by ghost** [comment]
I'm also facing the same error, hope it fixes soon

**Comment by stealthrabbi** [comment]
I tried updating the cookie.txt with a new cookie, but that doesn't work.

**Comment by vra** [comment]
Hi @stealthrabbi @Anaxyjod ,  It seems an issue about network accessing, could you visit https://www.bing.com/images/create in a browser?

**Comment by stealthrabbi** [comment]
Yes, I can visit the page in a browser and generate images. Generating a new cookie sometimes works, but stops working again after a few days.

**Comment by Supa24** [comment]
> I am currently getting this error when trying to use Bing Brush. I have Debug logging on but not many details.
> 
> ```
> 2024-01-26 18:12:30,252 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): www.bing.com:443
> 2024-01-26 18:12:30,538 - urllib3.connectionpool - DEBUG - https://www.bing.com:443 "POST /images/create?q=realistic%20Barney%20the%20Dinosaur%20Scuba%20Diving%20in%20a%20barrier%20reef%0A&rt=4&FORM=GENCRE HTTP/1.1" 200 129189
> ==> Generating...
> ==> Your boosts have run out, using the slow generating pipeline, please wait...
> 2024-01-26 18:12:30,725 - urllib3.connectionpool - DEBUG - https://www.bing.com:443 "POST /images/create?q=realistic%20Barney%20the%20Dinosaur%20Scuba%20Diving%20in%20a%20barrier%20reef%0A&rt=3&FORM=GENCRE HTTP/1.1" 200 129145
> ==> Error occurs, please submit an issue at https://github.com/vra/bing_brush, I will fix it as soon as possible.
> ```

try to update python to 3.12.2

**Comment by stealthrabbi** [comment]
I don't see how updating the version of python will help.

**Comment by stealthrabbi** [comment]
Putting in a new cookie temporarily resolves the problem. But if I just leave my existing cookie in place, it may not work for a day or two and eventually starts working again. But putting in a new cookie gets it working again right away, but the new cookie will stop working after some time, likely.

**Comment by AllFatherAI** [comment]
Hey @stealthrabbi  I've created a PR with the fix. Please give it a try: https://github.com/vra/bing_brush/pull/4/files#diff-e6759c5cd62a87bc418d56272a90976a8d9742f836fe81ef3707eda597eec462R60

**Comment by stealthrabbi** [comment]
@AllFatherAI I'm using the built library with my package manger, so I'm not sure how to try something on a development branch like this.

**Comment by AKwoKWH** [comment]
> @AllFatherAI I'm using the built library with my package manger, so I'm not sure how to try something on a development branch like this.

I think you can simply install the latest now, it is merged into the main branch
